devicetypes/fibargroup/fibaro-wall-plug-usb.src/fibaro-wall-plug-usb.groovy

```diff
@@ -0,0 +1,53 @@
+/**
+ *	Fibaro Wall Plug US child
+ */
+metadata {
+	definition (name: "Fibaro Wall Plug USB", namespace: "FibarGroup", author: "Fibar Group") {
```

Comment:
If this device is a switch, it needs the Switch capability. It should probably also have Actuator for completeness' sake.

However, this child device DTH could be removed and the parent device could just instantiate this DTH, since it seems to be the intended functionality anyway:

https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/fibargroup/fibaro-double-switch-2-usb.src/fibaro-double-switch-2-usb.groovy

Reply:
@greens  The USB device is not a switch therefore you cannot turn it on or off.  It seems to be logical to have the child DTH separate from the parent DTH because it has its own meter values.